app/Config/acl.php

```diff
@@ -124,22 +124,22 @@
  * the roles defined in your role configuration.
  */
 $config['alias'] = array(
-	'Role/4' => 'Role/editor',
+    'Role/4' => 'Role/editor',
 );
 
 /**
  * role configuration
  */
 $config['roles'] = array(
-	'Role/admin' => null,
+    'Role/admin' => null,
 );
 
 /**
  * rule configuration
  */
 $config['rules'] = array(
-	'allow' => array(
-		'*' => 'Role/admin',
-	),
-	'deny' => array(),
+    'allow' => array(
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 20 spaces but found 4
* Multi-line array contains a single value; use single-line array instead